RESideMenu/RESideMenuController.m

```diff
@@ -711,6 +714,13 @@ - (void)resetContentViewScale
     self.contentViewContainer.frame = frame;
 }
 
+- (void)performInitialAppearanceTransitionCallsForControllerIfRequired:(UIViewController *)controller {
+    if (controller.childViewControllers.count > 0) {
+        [controller beginAppearanceTransition:YES animated:NO];
```

Comment:
Good point on the missing disappear call, I'll add that in.
I tried doing this approach when the menu was about to be shown the first time, but then when the menu actually showed, the top layout guide was set to 44, which looks like the navigation bar height without the status bar taken in to consideration. `viewDidLoad` seems like the easiest place to make sure that everything is configured properly before the first real appearance.